<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackManagerPanel</class>
 <widget class="QWidget" name="BackManagerPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BackManagerPanel</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>261</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
   </property>
   <item>
    <property name="text">
     <string>机构信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>绩效评级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>查看</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>设置</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_name_team_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>金融机构绩效考核系统-------后台管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>911</width>
     <height>711</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>887</width>
          <height>687</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>机构信息</string>
           </property>
           <widget class="QLineEdit" name="lineEdit_search">
            <property name="geometry">
             <rect>
              <x>590</x>
              <y>26</y>
              <width>241</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>搜索机构相关信息</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>530</x>
              <y>30</y>
              <width>58</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
           <widget class="QTableWidget" name="tableWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>841</width>
              <height>541</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>机构编号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>机构名称</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>金融服务类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>地址</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>公司法人</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>状态</string>
             </property>
            </column>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>866</width>
          <height>1200</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1200</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>绩效评级</string>
           </property>
           <widget class="QTableWidget" name="tableWidget_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>821</width>
              <height>541</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>机构编号</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>机构名称</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>金融服务类型</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>地址</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>公司法人</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>状态</string>
             </property>
            </column>
           </widget>
           <widget class="QLabel" name="label_name_14">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>金融机构数据表：</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>810</y>
              <width>811</width>
              <height>111</height>
             </rect>
            </property>
            <property name="title">
             <string>辅助（该决策仅提供参考）</string>
            </property>
            <widget class="QPushButton" name="pushButton_help_me_decision_level">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>40</y>
               <width>201</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>帮我决策绩效评级</string>
             </property>
            </widget>
           </widget>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>650</y>
              <width>811</width>
              <height>141</height>
             </rect>
            </property>
            <property name="title">
             <string>功能</string>
            </property>
            <widget class="QPushButton" name="pushButton_performanceAll">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>40</y>
               <width>151</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>向全体发起绩效考核</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_performance_rules">
             <property name="geometry">
              <rect>
               <x>270</x>
               <y>40</y>
               <width>211</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>绩效评级指标及考核样式表</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_see_performanceResult">
             <property name="geometry">
              <rect>
               <x>580</x>
               <y>40</y>
               <width>151</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>查看绩效结果</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_3">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea_3">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>887</width>
          <height>687</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>查看</string>
           </property>
           <widget class="QPushButton" name="pushButton_fresh">
            <property name="geometry">
             <rect>
              <x>760</x>
              <y>20</y>
              <width>93</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
           <widget class="QListWidget" name="listWidget_see">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>851</width>
              <height>591</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_4">
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea_4">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>887</width>
          <height>687</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>设置</string>
           </property>
           <widget class="QLineEdit" name="lineEdit_new_passwd">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>100</y>
              <width>301</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_old_passwd">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>30</y>
              <width>301</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_129">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>170</y>
              <width>91</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>确认新密码</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_128">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>100</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>新密码</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_new_passwd_again">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>170</y>
              <width>301</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_90">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>30</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>旧密码</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_alter_passwd">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>260</y>
              <width>101</width>
              <height>41</height>
             </rect>
            </property>
            <property name="text">
             <string>修改</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
